client/extensions/woocommerce/app/shipping/shipping-card.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import FormCheckbox from 'components/forms/form-checkbox';
+import PaymentLogo from 'components/payment-logo';
+
+class ShippingCard extends Component {
+	render() {
+		const { type, digits, name, date } = this.props;
+		const __ = i18n.translate;
+
+		return (
+			<CompactCard className="shipping__card">
+				<FormCheckbox
+					className="shipping__card-checkbox"
+				/>
+				<div className="shipping__card-info">
```

Comment:
I just discovered the [`StoredCard` component](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/checkout/stored-card.jsx) - should we be using that here? (cc: @kellychoffman)